cmd/neofs-cli/modules/container/create.go

```diff
@@ -129,6 +129,7 @@ It will be stored in sidechain when inner ring will accepts it.`,
 				select {
 				case <-ctx.Done():
 					common.ExitOnErr(cmd, "", errCreateTimeout)
```

Comment:
this statement and `errCreateTimeout` variable are no longer needed because u add the next exiting instruction